packages/good-design/src/stories/apps/magik/Magik.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import MagikWidget from "@magiklabs/react-sdk";
+
+export const MagikWidgetOfframp = {
+  args: {
+    isProdUrl: true,
+    defaultCountry: "NG",
+    enableDebugLogs: false,
+    environment: "staging",
```

Comment:
why is env "staging" ?

Reply:
Env can be `staging` or `production` - for testing in stories, I set it to staging. Feel free to change to production if you'd like.